api/v1beta2/ibmpowervsmachine_webhook.go

```diff
@@ -70,10 +70,7 @@ func (r *IBMPowerVSMachine) ValidateDelete() error {
 
 func (r *IBMPowerVSMachine) validateIBMPowerVSMachine() error {
 	var allErrs field.ErrorList
-	if err := r.validateIBMPowerVSMachineSysType(); err != nil {
-		allErrs = append(allErrs, err)
-	}
-	if err := r.validateIBMPowerVSMachineProcType(); err != nil {
```

Comment:
is this removed because its not required anymore because of the enum?

Reply:
Yes.